.github/workflows/build.yml

```diff
@@ -2,27 +2,41 @@
 name: Build images and push to quay.io
 
 on:
+  # Start build when GitHub release is published
   release:
     types: [published]
+  # Allow manual trigger
+  workflow_dispatch:
+  # Rebuild every Monday at 4:30 UTC
+  schedule:
+    - cron: '30 4 * * 1'
 
 jobs:
-  # this job will only be triggered when release is published
+  # this job will only be triggered when on of the above trigger condition is met
   deployments:
     name: Build and Push to quay.io
     runs-on: ubuntu-latest
 
     steps:
+      - name: Get latest tag
```

Comment:
Does this mean any new changes will be added to the last released tag?

Reply:
This will get name of the latest tag which was released on (GitHub). 
That is set by `releases-only: true` and to avoid selecting other tags it should have a prefix `v` as we are used to set for our version e.g. `v6.7.2`

Reply:
Aha, makes sense. thank you!